lib/filepicker_rails/policy.rb

```diff
@@ -1,6 +1,44 @@
 require 'base64'
 require 'openssl'
 
+# Generate a policy and signature from the given options
+#
+# #### Constructor
+#
+# @param [Hash] options
```

Comment:
This way no documentation is generated for me, you need to put what is related withe initialize above the initialize like:

```
    # @param [Hash] options
    #
    def initialize(options = {})
 ```